.pre-commit-config.yaml

```diff
@@ -18,9 +18,9 @@ repos:
           - yaml
 
   - repo: https://github.com/omnilib/ufmt
-    rev: v1.3.2
+    rev: v2.8.0
```

Comment:
Probably want to switch to `ruff` in the near future.

Reply:
Absolutely! I'll make this switch sometime in the future. We have been using `ruff` in the Slicer project (see [here](https://github.com/Slicer/Slicer/blob/1af67ed6f1411d9dac9f8d7fe246469bb0556569/.pre-commit-config.yaml#L20-L24)) for a little over a year now. It has been a great package to use.